src/components/QuorumBar/QuorumBar.js

```diff
@@ -15,14 +17,23 @@ const QuorumBar = ({ totalVoters, quorum, actualVoters, proposalStatus }) => {
   if (proposalStatus === PROPOSAL_STATUS_COMPLETED && !quorumReached) quorumStatus = t('Quorum not reached')
 
   return (
-    <div styleName='vote-progress-container'>
-      <div styleName='actual-voters' style={{ width: `${actualVotersWidth}%` }}>
-        <div styleName='quorum-text'>{quorumStatus}</div>
+    <div styleName='quorum-container'>
+      <Icon name='Info' styleName='quorum-tooltip' dataTip={t('quorumExplainer')} dataTipFor='quorum-tt' />
+      <div styleName='vote-progress-container'>
+        <div styleName='actual-voters' style={{ width: `${actualVotersWidth}%` }}>
+          {quorum > 10 && <div styleName='quorum-text'>{quorumStatus}{' '}{quorumReached && quorum > 20 && t('voterCount', { count: actualVoters })}</div>}
+        </div>
+        <div styleName='quorum-bar' style={{ width: `${quorum}%` }}>
+          {!quorumReached && <div styleName={cx('quorum-number', { 'quorum-reached': quorumReached, 'big-quorum': quorum > 70 })}>{quorum}% {actualVoters || 0}/{votersForQuorum}</div>}
```

Comment:
I still think we will want to tweak this. at least put the voter #s in parentheses, but probably have it say something like 0 voters out of 10 needed. but we can merge this now